src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/BatchExecuteWithErrorsTest.java

```diff
@@ -473,4 +695,208 @@ public void Repro47239large() throws Exception {
         stmt.close();
         conn.close();
     }
+    
+    /**
+     * Tests large methods, supported in 42
+     * 
+     * @throws Exception
+     */
+    @Test
```

Comment:
Duplicate of `Repro47239large`. Please refactor.

Reply:
Done.